frontend/public/components/masthead-toolbar.jsx

```diff
@@ -222,26 +216,33 @@ class MastheadToolbar_ extends React.Component {
   }
 
   _launchActions = () => {
-    const { consoleLinks } = this.props;
+    const { clusterID, consoleLinks } = this.props;
     const launcherItems = this._getAdditionalLinks(consoleLinks, 'ApplicationMenu');
 
-    const redHatApplications = {
-      name: 'Red Hat Applications',
-      isSection: true,
-      actions: [multiClusterManager],
-    };
+    const sections = [];
+    if (clusterID) {
+      sections.push({
+        name: 'Red Hat Applications',
+        isSection: true,
+        actions: [{
+          label: 'OpenShift Cluster Manager',
+          externalLink: true,
+          href: getOCMLink(clusterID),
+          image: <img src={openshiftLogoImg} alt="" />,
+        }],
+      });
+    }
 
-    const sections = _.reduce(launcherItems, (accumulator, item) => {
+    _.each(launcherItems, (item) => {
       const sectionName = _.get(item, 'spec.applicationMenu.section', '');
-      if (!_.find(accumulator, {name: sectionName})) {
-        accumulator.push({name: sectionName, isSection: true, actions: []});
+      if (!_.find(sections, {name: sectionName})) {
+        sections.push({name: sectionName, isSection: true, actions: []});
       }
-      return accumulator;
-    }, [redHatApplications]);
+    });
 
-    _.sortBy(sections, [this._sectionSort, 'name']);
```

Comment:
This looks like it was a bug... `_.sortBy` returns a new array instead of sorting in place.

https://lodash.com/docs/4.17.15#sortBy